docs/openapi.md

```
@@ -62,11 +62,14 @@ Lightspeed Core Service (LCS) service API specification.
 
 Handle GET requests to the root ("/") endpoint and returns the static HTML index page.
 
-Returns:
+### Parameters:
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)